learn/avalanche-bridge-faq.md

```diff
@@ -46,6 +46,12 @@ Yes, you can click on the “Speed Up” button on Metamask. “Speeding up” a
 
 When transferring from Avalanche to Ethereum, Metamask shows that 0 tokens are to be transferred, not the actual number of tokens. This is a known problem with Metamask.
 
+#### What are the bridge contract adresses? 
+
+Bridge Addresses:
```

Comment:
Can we add a note here that we don't officially support direct transfers (ie not using the frontend) to these addresses.  And that we're not responsible for lost funds due to transfers under the minimal requirements (the frontend prevents this)

Reply:
@salwao2 can you address Connor's comments in this PR?